scripts/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "google"
+    ],
+    "globals": {
+        "Atomics": "readonly",
```

Comment:
You don't used any 'special abilities' of JS especially  Atomics, SharedArrayBuffer so do not expect them to be in a global scope.